src/components/Modal/Modal/Modal.js

```diff
@@ -0,0 +1,17 @@
+import { IoClose } from "react-icons/io5";
+import "./Modal.css";
+import { Fragment } from "react";
+
+export const Modal = ({ openModal, children, handleModalClose, modalRef }) => {
+  return (
+    <Fragment>
+      <div className="modal-overlay"></div>
+      <div ref={modalRef} className={openModal ? "modal open" : "modal"}>
+        <div onClick={handleModalClose}>
+          <IoClose className="modal-closeBtn" />
+        </div>
```

Comment:
P3:
button tag 추천드려요.
```suggestion
        <button type='button' onClick={handleModalClose}>
          <IoClose className="modal-closeBtn" />
        </button>
```